image/config.go

```diff
@@ -73,12 +73,92 @@ func (c *config) runtimeSpec(rootfs string) (*specs.Spec, error) {
 		return nil, fmt.Errorf("%s: unsupported OS", c.OS)
 	}
 
-	var s specs.Spec
-	s.Version = specs.Version
-	// we should at least apply the default spec, otherwise this is totally useless
-	s.Process.Terminal = true
+	// TODO: Get this default from somewhere, rather than hardcoding it here.
```

Comment:
I suggest you get the default [from a vendored `runtime-tools/generate`][1].

[1]: https://github.com/opencontainers/runtime-tools/blob/bb040483d65d8731a2589f5d20c8b2367323219d/generate/generate.go#L34